### DISCUSSION TITLE
DAB+ stopped working after update to 20221117 and upgrade

### DISCUSSION THREAD

**Discussion by skycamel** [rern/rAudio#1102] [Fixed Issues]
When I try DAB+ in terminal it is working see;

rAudio:~  #  dab-rtlsdr-3 -C 12A -S 6305 -i /tmp >/tmp/radio.pcm
dab_cmdline example III,
 	                  Copyright 2017 J van Katwijk, Lazy Chair Computing
options are i:T:D:d:M:B:P:O:A:C:G:p:Q:S:
going for rtlsdr 223936000 50
OK, functions seem to be loaded
Found Rafael Micro R820T/2 tuner
samplerate set to 2048000
Supported gain values (29): 0.0 0.9 1.4 2.7 3.7 7.7 8.7 12.5 14.4 15.7 16.6 19.7 20.7 22.9 25.4 28.0 29.7 32.8 33.8 36.4 37.2 38.6 40.2 42.1 43.4 43.9 44.5 48.0 49.6 
effective gain: gain 25.4
Allocating 15 zero-copy buffers
there might be a DAB signal here
ensemble DAB+ VRT         is (6001) recognized
radio2 lim       (6602) is part of the ensemble
Klara            (6303) is part of the ensemble
Radio 1          (6301) is part of the ensemble
radio2 o-vl      (6C02) is part of the ensemble
radio2 ant       (6502) is part of the ensemble
radio2 w-vl      (6B02) is part of the ensemble
Studio Brussel   (6304) is part of the ensemble
TPEG             (data) (E0600001) is part of the ensemble
MNM hits         (631F) is part of the ensemble
radio2 vl-br     (6402) is part of the ensemble
VRT NWS          (631E) is part of the ensemble
radio2 benebene  (631B) is part of the ensemble
Klara Continuo   (631A) is part of the ensemble
MNM              (6305) is part of the ensemble
StuBru Tijdloze  (631C) is part of the ensemble

we try to start program MNM             
protection handler is eep_protection
we have now DAB+
	startaddress	= 540
	length		= 66
	subChId		= 9
	protection	= 2
	bitrate		= 88
^CTEJOOR feat. BABET - 1 OP EEN MILJOEN

But when I try to start a DAB+ channel in the GUI then it will stop after a few seconds (and no sound)
The MDP gives following error
Nov 19 16:20:15 rAudio mpd[4489]: player: played "rtsp://rAudio:8554/11a_6397"
Nov 19 16:20:18 rAudio mpd[4489]: ffmpeg/rtsp: method DESCRIBE failed: 404 Not Found
Nov 19 16:20:18 rAudio mpd[4489]: exception: Failed to decode rtsp://rAudio:8554/11a_6397; avformat_open_input() failed: Server returned 404 Not Found

Any help please?
Luc

**Comment by kan-ibal** [comment]
You have tried different channel and station in terminal then the one form GUI. Check "-C 11A -S 6397" instead. Have you got DAB Radio enabled in Settings ->Features? It auto disables when crashes and that happens very often.

**Comment by skycamel** [comment]
Got it working again, luckily :-)
This time I did the update to 20221117 first and then I did the upgrade. (before I did the other way around)
Still then it wasn't working, but after re-scanning the DAB+ everything is workingagain.
Cheers,
Luc
